src/components/Avatar/Avatar.scss

```diff
@@ -0,0 +1,20 @@
+@import "~styles/main";
+
+.container {
+  img {
+    border-radius: 50%;
+    width: 50px;
+    height: 50px; //70px
+    margin: 0px 0px;
+  }
+}
+.dotted {
+  img {
+    border: 2px dashed black; // add logic to make this line conditional
```

Comment:
Problem: We can't use conditional statements in CSS.
Suggestion: We can use modifiers (check out BEM), then implement a conditional statement on JS to specify modifiers for this particular problem.